### ISSUE TITLE
Multiple edit windows for comparison/copying.

### ISSUE THREAD

**Issue by Lilu-VA** [BLCM/OpenBLCMM#67]
It would be useful to be able to open multiple of the edit window and interact with the main BLCMM window while an edit is open.

**Comment by apocalyptech** [comment]
Yeah, I agree it would be useful but I wouldn't expect it anytime soon.  The only reasonable thing to do would be to ensure that the main window goes into a readonly state while the edit window's open, and there's currently no real provision for something like that.  The actual base Component for the edit window might need to change, too, though I could look into that.

Anyway, yeah, will keep this open, but it's probably pretty low on my own priorities.  Cheers, thanks for the report!
